src/style.css

```diff
@@ -0,0 +1,106 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+}
+
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.box {
+  height: 300px;
+  color: #fff;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  flex-basis: 300px;
+  flex-grow: 1;
+  flex-shrink: 0;
+}
+
+.box-1 {
+  background-color: rgb(255, 0, 0);
+}
+
+.box-2 {
+  background-color: rgb(204, 0, 0);
+}
+
+.box-3 {
+  background-color: rgb(153, 0, 0);
+}
+
+.box-4 {
+  background-color: rgb(102, 0, 0);
+}
+
+.box-5 {
+  background-color: rgb(51, 0, 0);
+}
+
+.box-6 {
+  background-color: rgb(0, 0, 0);
+}
+
+@media (min-width: 600px) and (max-width: 899px) {
+  .box-3 {
+    order: 2;
+  }
+
+  .box-4 {
+    order: 1;
+  }
+
+  .box-5 {
+    order: 3;
+  }
+
+  .box-6 {
+    order: 4;
+  }
+}
+
+@media (min-width: 900px) and (max-width: 1200px) {
+  .box-4 {
+    order: 3;
+  }
+
+  .box-5 {
+    order: 2;
+  }
+
+  .box-6 {
+    order: 1;
+  }
+}
+
+@media (min-width: 1201px) and (max-width: 1499px) {
+  .box-5 {
+    order: 2;
+  }
+
+  .box-6 {
+    order: 1;
+  }
+}
+
+@media (min-width: 1500px) {
+  .box-5 {
+    order: 1;
+  }
+
+  .box-6 {
+    order: 2;
+  }
+}
```

Comment:
delete, should be the same order as on 900px, and each block should have a width of 33.3%